spec/support/time_helper.rb

```diff
@@ -0,0 +1,5 @@
+require "active_support/testing/time_helpers"
+
+RSpec.configure do |config|
+  config.include ActiveSupport::Testing::TimeHelpers
+end
```

Comment:
This is a RSpec way to include a module (module has methods) to all the describe/context/it blocks

https://relishapp.com/rspec/rspec-core/docs/helper-methods/define-helper-methods-in-a-module